### PR TITLE
feat/gh actions pipelines

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

File: .github/workflows/prod-docker.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          release-type: node
+          package-name: ""
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"ci","section":"CI / CD","hidden":false},{"type":"test","section":"Testing","hidden":false},{"type":"refactor","section":"Refactorings","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+
+      - name: Set up Docker Buildx
+        id: buildx
+        # Use the action from the master, as we've seen some inconsistencies with @v1
+        # Issue: https://github.com/docker/build-push-action/issues/286
+        uses: docker/setup-buildx-action@master
+        # Only worked for us with this option on �‍♂️
```
Comment (venimus) [inline .github/workflows/prod-docker.yml:36]: some junk landed here

File: .github/workflows/prod-docker.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+on:
+  push:
+    branches:
+      - master
+    release:
+      types: [created]
+
+env:
+  ENV: "prd"
```
Comment (venimus) [inline .github/workflows/prod-docker.yml:11]: are these prd/stg really named that way?

Comment (mmpetarpeshev) [reply .github/workflows/prod-docker.yml:11]: Yes , they are named like that , thats what we decided with Dincho . It matches the gitops branches, etc..We can discussed it , if you have something in mind .

File: .github/workflows/prod-docker.yml (hunk 1 above)
Comment (mmpetarpeshev) [reply .github/workflows/prod-docker.yml:36]: will fix

File: .github/workflows/prod-docker.yml (hunk 2 above)
Comment (venimus) [reply .github/workflows/prod-docker.yml:11]: not really, just unusual

**Review by venimus — APPROVED** [review]

**Review by dincho — APPROVED** [review]

**Comment by marc0olo** [comment]
is this project still relevant? we did re-implementation in js:
- https://github.com/aeternity/aepp-faucet-nodejs

**Comment by mmpetarpeshev** [comment]
I don't know , it was in our list of projects for migration to eks . Do we need gh pipelines for https://github.com/aeternity/aepp-faucet-nodejs ? There is some initial one , but we can apply the workflow from other repositories like dex-ui.

**Comment by marc0olo** [comment]
yes. in general we should replace the deployment of this repository with the nodejs based implementation if possible

**Comment by marc0olo** [comment]
if the deployment of nodejs based faucet is actually in place we can archive this repo here

**Pull request merged**